### ISSUE TITLE
Redundant method

### ISSUE THREAD

**Issue by sen1** [bahobab/TomeRater#5]
https://github.com/bahobab/TomeRater/blob/c8420e0107e5762b9a4e7791c1ef722427b605ba/TomeRater.py#L90-L102

This should not be part of `Book` class. As soon as a book object created, the `isbn` is assigned in the `__init__` method. Follow my last comments on an earlier issue why this helper function should be removed.

**Comment by bahobab** [comment]
Yes. I wanted to make sure that 2 books don't have the same ISBN. But looks like this is not a requirement, for 2 books are equal if they have same ISBN AND name
